pkg/policy/groups/actions.go

```diff
@@ -225,33 +225,25 @@ func addDerivativePolicy(ctx context.Context, cnp *cilium_v2.CiliumNetworkPolicy
 		})
 	}
 
-	// The maxNumberOfAttempts is to not hit the limits of cloud providers API.
-	// Also, the derivativeErr is never returned, if not the controller will
-	// hit this function and the cloud providers limit will be raised. This
-	// will cause a disaster, due all other policies will hit the limit as
-	// well.
 	// If the createDerivativeCNP() fails, a new all block rule will be inserted and
 	// the derivative status in the parent policy  will be updated with the
 	// error.
-	for numAttempts := 0; numAttempts <= maxNumberOfAttempts; numAttempts++ {
-		if clusterScoped {
-			derivativeCCNP, derivativeErr = createDerivativeCCNP(ctx, cnp)
-			derivativePolicy = derivativeCCNP
-		} else {
-			derivativeCNP, derivativeErr = createDerivativeCNP(ctx, cnp)
-			derivativePolicy = derivativeCNP
-		}
+	if clusterScoped {
```

Comment:
The `maxNumberOfAttempts` can be removed from the top of this file as well.

Reply:
Done.